lib/plugins/aws/deploy/compile/events/apiGateway/lib/deployment.js

```diff
@@ -37,9 +38,11 @@ module.exports = {
       ServiceEndpoint: JSON.parse(outputServiceEndpointTemplate),
     };
 
-    this.serverless.service.resources.Outputs =
-      this.serverless.service.resources.Outputs || {};
-    _.merge(this.serverless.service.resources.Outputs, newOutputEndpointObject);
+    this.serverless.service.provider.compiledCloudFormationTemplate.Outputs =
```

Comment:
Don't think this line is necessary any more as the core template will create Outputs already